metrics/grafana/pd.json

```diff
@@ -134,7 +134,7 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "pd_tso_role{tidb_cluster=\"$tidb_cluster\", instance=\"$instance\", dc=\"global\"}",
+          "expr": "pd_tso_role{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=\"$instance\", dc=\"global\"}",
```

Comment:
Will it cause tiup cannot display?

Reply:
If `k8s_cluster` is empty, then the condition will be like `k8s_cluster=""`, which is the same as a no op in prometheus.